docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -73,9 +78,9 @@ Example of using ``InputCutInfo`` to freeze an input with value:
 .. code-block:: py
    :force:
 
-   from openvino.tools.mo import convert_model, InputCutInfo
-   
-   ov_model = convert_model(model, input=InputCutInfo("input_name", [3], np.float32, [0.5, 2.1, 3.4]))
+   import openvino as ov
+
+   ov_model = ov.convert_model(model, input=ov.InputCutInfo("input_name", [3], np.float32, [0.5, 2.1, 3.4]))
```

Comment:
```suggestion
   from openvino.tools.ovc import InputCutInfo

   ov_model = ov.convert_model(model, input=InputCutInfo("input_name", [3], np.float32, [0.5, 2.1, 3.4]))
```
I think you need to import it directly.

Reply:
This will not work, we removed InputCutInfo from ovc.
Please import it from openvino.tools.mo